distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -256,6 +260,10 @@ grant codeBase "file:/admin-core-insecuredefaults/org.apache.felix.scr/org.apach
     permission java.io.FilePermission "${ddf.home.perm}etc${/}artemis.xml", "read";
 }
 
+grant codeBase "file:/admin-core-insecuredefaults" {
```

Comment:
❓ Is this just the one that deletes it (or rather blanks out the file)?

Reply:
Blanks out the file